### ISSUE TITLE
The named parameter 'transition' isn't defined. Try correcting the name to an existing named parameter's name, or defining a named parameter with the name 'transition'.

### ISSUE THREAD

**Issue by shayant98** [Stacked-Org/stacked#644]
after updating stacked_builder to 0.7.1, it added a transition value to the NavigatorStateExtension. 
this is passed down to a generated function which, does not have a transition parameter

```dart
Future<dynamic> navigateToStartupView({
    int? id,
    bool preventDuplicates = true,
    Map<String, String>? parameters,
    Widget Function(BuildContext, Animation<double>, Animation<double>, Widget)?
        transition,
  }) async {
    return navigateTo(
      Routes.startupView,
      id: id,
      preventDuplicates: preventDuplicates,
      parameters: parameters,
      transition: transition,
    );
  }
```

``` dart
  /// Pushes [routeName] onto the navigation stack
  ///
  /// [id] is for when you are using nested navigation, as explained in documentation.
  ///
  /// [preventDuplicates] will prevent you from pushing a route that you already in, if you want to push anyway,
  /// set to false.
  Future<T?>? navigateTo<T>(
    String routeName, {
    dynamic arguments,
    int? id,
    bool preventDuplicates = true,
    Map<String, String>? parameters,
  }) {
    return G.Get.toNamed<T?>(
      routeName,
      arguments: arguments,
      id: id,
      preventDuplicates: preventDuplicates,
      parameters: parameters,
    );
  }
```

**Comment by Nirajn2311** [comment]
This has been fixed in `stacked_services: 0.8.26`

**Comment by shayant98** [comment]
that's great to hear! i'll close this then.
